src/components/TokenSelector.tsx

```diff
@@ -262,16 +288,15 @@ const TokenSelector: React.FC<Props> = ({ isDisabled, tokens, selected, onChange
         classNamePrefix="react-select"
         noOptionsMessage={(): string => 'No results'}
         formatOptionLabel={formatOptionLabel}
-        // menuIsOpen={true}  // uncomment to be able to interact with menu styles
         options={options}
         value={{ token: selected }}
-        onChange={(selected: { token: TokenDetails }, { action }: ActionMeta): void => {
-          if (selected && action === 'select-option' && 'token' in selected) {
-            onChange(selected.token)
-          }
-        }}
+        onChange={onSelectChange}
         tabIndex={tabIndex.toString()}
-        components={{ MenuList: menuListFactory() }}
+        components={{ MenuList }}
+        isFocused={isFocused || undefined}
+        menuIsOpen={isFocused || undefined} // set to `true` to make it permanently open and work with styles
```

Comment:
why `|| undefined`? shouldn't be necessary

Reply:
Because when these values are set to either `true`/`false`, we are assuming full control over those properties. When `undefined`, the control is handled back to `react-select`.

Reply:
Got it :+1: